client/server.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
As a private server key do we want this to be on our public github?

Reply:
It's the self-signed certificate, so it's fine to keep it on github. The certificates signed by trusted authorities are the ones we need to keep off github

Reply:
> @alir128 Do we need to merge this to master? Because this would break the client for the aws setup rn. We need this only for heroku, so let's keep it on the heroku branch until we fully transition to heroku?

@Lakshya2610 if we don’t add this to master, for anyone that might want to extend the project or just wants to clone and use locally will encounter issues and the vue client will not work for them over https. AWS and heroku both are using Signed certs so we dont have that issue over there but for this vue client to work for other people I think we should merge it.